### ISSUE TITLE
Latte-dock-git does not install

### ISSUE THREAD

**Issue by Zahrun** [pacstall/pacstall-programs#2024]
@wizard-28 
It seems latte-dock-git does not build anything anymore

**Comment by Zahrun** [comment]
Thank you for the commit.
I still get missing dependencies, it seems [latte-dock.log](https://github.com/pacstall/pacstall-programs/files/10320256/latte-dock.log)

**Comment by wizard-28** [comment]
> Thank you for the commit. I still get missing dependencies, it seems [latte-dock.log](https://github.com/pacstall/pacstall-programs/files/10320256/latte-dock.log)

Does latte dock work?

**Comment by Zahrun** [comment]
No it’s not building it seems

**Comment by Henryws** [comment]
Based on the log you sent, it is building and installing.

**Comment by wizard-28** [comment]
The installation was successful, as per your logs

**Comment by Zahrun** [comment]
It installs without building, then I get `command not found: latte-dock`
![image](https://user-images.githubusercontent.com/10415894/209980810-4c15bfcc-7da7-4c6d-94f1-9329b6d87344.png)

**Comment by Henryws** [comment]
Why doesn't cmake fail if the dependencies weren't satisfied?

**Comment by Zahrun** [comment]
What is this in the logs?
```
	[[1;34m>[0m] Running install
cat install.sh >install 
chmod a+x install
```

**Comment by Henryws** [comment]
Not sure

**Comment by wizard-28** [comment]
> [�[1;34m>�[0m] Running install
> cat install.sh >install 
> chmod a+x install

It's the ANSI color encoded text I believe, you can `cat` the log file, to see the colors

**Comment by Zahrun** [comment]
I meant more the two other lines. I found out the issue.
pacstall runs `install()` in the latte-dock git folder instead of inside thatSameFolder/build

**Comment by Zahrun** [comment]
```
:/tmp/latte-dock master ± sudo make install DESTDIR=target
cat install.sh >install 
chmod a+x install
```

vs

```
/tmp/latte-dock/build master ± sudo make install DESTDIR=target
[  1%] Generating mo...
[  1%] Built target pofiles-25ec4fa72206eab41621582ad7d80ee6
[  2%] Generating ts...
[  2%] Built target tsfiles-25ec4fa72206eab41621582ad7d80ee6
```

**Comment by wizard-28** [comment]
> I meant more the two other lines. I found out the issue. pacstall runs `install()` in the latte-dock git folder instead of inside thatSameFolder/build

so `cd`-ing into `build/` fixes it?

**Comment by Zahrun** [comment]
Yes. I remember we had a similar issue in the 3.0.0 pacstall version

**Comment by Zahrun** [comment]
cf. https://github.com/pacstall/pacstall-programs/pull/1682

**Comment by wizard-28** [comment]
Okay, so this seems like a pacstall bug, @Henryws is working on a fix.

**Comment by Henryws** [comment]
I added a `cd build`/ to `install()`

**Comment by Zahrun** [comment]
> I added a `cd build`/ to `install()`

So the solution is to change all pacscript which needs to stay in the same folder to use cd? Other packages are affected like obs-backgroundremoval

**Comment by Zahrun** [comment]
@Henryws cf. https://github.com/pacstall/pacstall-programs/pull/2101 , #2102 
Other packages might also have been broken by the change
